Composer/packages/electron-server/electron-builder-config.json

```diff
@@ -15,15 +15,10 @@
     "build/templates",
     "resources/composerIcon_1024x1024.png",
     "resources/ms_logo.svg",
-    "**/node_modules/orchestrator-core/**/*",
+    "**/node_modules/@microsoft/orchestrator-core/**/*",
```

Comment:
this is the only change - the rest of changes appear to be whitespace changes by linter upon checkin.